### ISSUE TITLE
Contact synching

### ISSUE THREAD

**Issue by ChrisB0701** [danielcardeenas/whatsapp-framework#67]
This is not an issue but a general question: If my bot receives a message from a contact that is not listed in the "contact" structure in config.py, does the framework synch the new contact automatically? If not, how can I synch the contact manually?

Thanks!

**Comment by SPWC** [comment]
My contacts structure doesn't populate at all, also wondering if there is something I'm doing wrong

**Comment by danielcardeenas** [comment]
Yeah this can be done manually. Give me one moment and I'll update it in the next push

**Comment by m7mdcc** [comment]
@danielcardeenas  any update on this ? 


Thanks mate !

**Comment by SPWC** [comment]
I'll look into this, unless anyone has already got something and cares to share??
Thinking it'll be a module function called to update  contact list

**Comment by ChrisB0701** [comment]
Any news on this? I could also execute the synch manually but I'm not sure which yowsup function to call. Can you help me on this? Thanks!

**Comment by SPWC** [comment]
Im also waiting for this function, would be nice to have an auto contact sync

**Comment by galihadbw** [comment]
yes I agree, if there is already automatic function to sync contacts. until now I'm still waiting.

thanks in advance for @danielcardeenas

**Comment by juhniorsantos** [comment]
I would like to filter the contacts that have whatsapp, is it possible today to do this?

**Comment by vjyanand** [comment]
AFAIK, there is no such method as of now
